src/schema/agency.ts

```diff
@@ -3,7 +3,7 @@ import { z } from "zod";
 
 export const agency = pgTable("agency", {
   initials: text("initials").primaryKey(),
-  name: text("name"),
+  name: text("name").notNull(),
```

Comment:
I think the status quo is that there are not, and never will be, any agencies without names.  We should make sure that the database does not allow for agencies without names, because it would imply there is an error with the data.  Were it possible for agencies not to have names, "initials" would not be an appropriate name for the primary key.

Reply:
Status quo from the perspective of the implementation. 

As stated above, there are arguments for both business perspectives. However, we do not have enough experience working with the data to understand which one is a more appropriate model of reality.

Reply:
Based on some discussions this sparked, some take aways:
- @TangoYankee we're thinking about failing loudly and how constraining the `name` type to just string allows us to do that
- I ran the drizzle migration that makes the change to allow `name` to _not_ be nullable 
- Closing the ticket https://github.com/NYCPlanning/ae-zoning-api/issues/290
- We'll go through each property of the new schemas to determine whether or now we want them to be nullable based on 1) if they currently have existing `NULL` values in the database and 2) reaching out to data-eng or product owners to determine whether they could be null in the future (i.e. submitting a project from a form that allows an empty description)

Reply:
Consider my mind changed